metrics/metrics-core/src/main/java/org/apache/servicecomb/metrics/core/event/InvocationStartProcessingEventListener.java

```diff
@@ -43,7 +43,6 @@ public void process(Event data) {
     if (InvocationType.PRODUCER.equals(event.getInvocationType())) {
       ProducerInvocationMonitor monitor = registryMonitor.getProducerInvocationMonitor(event.getOperationName());
       monitor.getWaitInQueue().increment(-1);
-      monitor.getLifeTimeInQueue().update(event.getInQueueNanoTime());
```

Comment:
Why did you remove this line?

Reply:
queue stage latency had move from this event to finish event